crates/stages/api/src/pipeline/mod.rs

```diff
@@ -420,6 +420,8 @@ where
                 };
             }
 
+            let provider_rw = self.provider_factory.provider_rw()?;
+
             self.event_sender.notify(PipelineEvent::Run {
```

Comment:
this is so we know whether or not acquiring the write tx is possibly stalled?

Reply:
correct